src/topology/algebra/module/finite_dimension.lean

```diff
@@ -366,6 +370,13 @@ by { ext x, refl }
   f.to_continuous_linear_equiv_of_det_ne_zero hf x = f x :=
 rfl
 
+lemma _root_.matrix.to_lin_prod_continuous_linear_map (a b c d : 𝕜) :
```

Comment:
I would argue this should be the more verbose
```suggestion
lemma _root_.matrix.to_lin_fin_two_prod_to_continuous_linear_map (a b c d : 𝕜) :
```
so that it shares a prefix with the other two `matrix.to_lin_fin_two_prod` lemmas